webhooks.md

```diff
@@ -28,10 +28,19 @@ When an action of particular type is performed in the system, a message is sent
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
 | `Action` | string [Webhook action](#webhook-action) | required | Type of the action. |
-| `Data` | object [Data](#integration-created-data) | required | Webhook data. |
+| `Data` | object [Data](#webhook-data) | required | Webhook data. |
 
 ## Webhook action
 
+* `IntegrationCreated`
+* `IntegrationEnabled`
+* `IntegrationDisabled`
+* `IntegrationCanceled`
+* `IntegrationReinstated`
+* `IntegrationDeleted`
+
+## Webhook data
+
 Structure of data varies depending on the webhook action:
```

Comment:
Now there should be probably a link to the tile above, no?:)

Reply:
But for me, it's duplicity with the section below, what is the purpose to have 2 times the same things?

I would you just the action part and then I would do:
```
## Webhook actions

- Integration created - triggered when a new integration created ([Integration created data](...))
- Integration enabled - triggered when integration is enabled ([Integration enabled data](...))
```
How about something like this?
cc @siroky

Reply:
Then, there would be no direct link from the table above to data, but that's fine I guess. You can put there description something like `Data | object | required | Structure of the object depends on [Webhook action].` And in "Webhook action" section have the table like you proposed. However the items should represent the values in JSON so `IntegrationCreated` instead of "Integration created". That would make sense to me and we can then remove the "Webhook data" section. I like this structure, please roll it out to all similar places as well, e.g. device command data...